From a9b0e2f094ce74d55661b84c50fb65a699c0f277 Mon Sep 17 00:00:00 2001
From: vinicius-mo <vinicius-mo at segplan.go.gov.br>
Date: Fri, 15 Mar 2019 13:50:10 -0300
Subject: [PATCH] gamemoderun: do not hardcode libdir in LD_PRELOAD

Set /usr/$LIB supported by glibc. This fixes multilib packaging
---
 data/gamemoderun.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/gamemoderun.in b/data/gamemoderun.in
index 9704d4d..2170892 100644
--- a/data/gamemoderun.in
+++ b/data/gamemoderun.in
@@ -2,7 +2,7 @@
 # Helper script to launch games with gamemode
 
 # Path to install gamemoded auto script
-CONFIG_LIB_DIR="@GAMEMODE_LIB_DIR@/libgamemodeauto.so.0"
+CONFIG_LIB_DIR="/usr/\$LIB/libgamemodeauto.so.0"
 
 # Set the ld preload path prefixed libgamemodeauto
 export LD_PRELOAD="${CONFIG_LIB_DIR}${LD_PRELOAD:+:$LD_PRELOAD}"
-- 
2.20.1

